src/components/DataDictionary.tsx

```diff
@@ -198,19 +198,12 @@ export const DataDictionary: React.FC<{}> = () => {
   const toggleFn = () => setSidebarOpen(!sidebarOpen)
 
   return (
-    <ThemeProvider theme={theme}>
-      <div style={{ minWidth: "1200px" }}>
+    <ComponentsProvider>
```

Comment:
```suggestion
    <ComponentsProvider themeCustomizations={{ colors: { key: '#1A73E8' } }}>
```

I'd rather we update the overall key color than just the header background.